tests/tasks/fivetran/test_fivetran.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import requests
+import responses
+import logging
+import os
+
+from prefect.task.fivetran import FivetranSyncTask
+
+logging.basicConfig()
+log = logging.getLogger()
+
+class TestFivetran:
+    def test_required_params(self):
+        """
+        Tests to check if there are missing required parameters.
+        """
+
+        # raises Value error if connector_id is not provided
+        with pytest.raises(ValueError, match="Value for parameter `connector_id` must be provided."):
+            FivetranSyncTask().run(
+                api_key="test",
+                api_secret="test",
+                schedule_type="auto",
+            )
+
+        # raises Value error if api_key is not provided
+        with pytest.raises(ValueError, match="Value for parameter `api_key` must be provided."):
+            FivetranSyncTask().run(
+                connector_id="test",
+                api_secret="test",
+                schedule_type="auto",
+            )
+        # raises Value error if api_secret is not provided
+        with pytest.raises(ValueError, match="Value for parameter `api_secret` must be provided."):
+            FivetranSyncTask().run(
+                connector_id="test",
+                api_key="test",
+                schedule_type="auto,"
+            ) 
+        # raises Value error if schedule_type is not "manual" or "auto"
+        with pytest.raises(ValueError, match='schedule_type must be either "manual" or "auto"'):
+            FivetranSyncTask().run(
+                connector_id="test",
+                api_key="test",
+                api_secret="test",
+                schedule_type="test",
+            )
+   
+    @pytest.mark.skipif(
+        os.getenv("FIVETRAN_TEST_API_KEY") is None,
+        reason="You need a Fivetran API Key and API Secret to perform an actual Fivetran test",
+    )
```

Comment:
We normally mock out API calls using `responses` instead of testing against live systems. You can look at the tests for the Airbyte or Databricks tasks for examples on how to mock API calls.